src/components/Copy/index.tsx

```diff
@@ -47,6 +46,7 @@ const BaseCopy = ({
 			{...otherProps}
 		>
 			{children}
+
```

Comment:
we can probably just revert the entire fille unless this is a change made by the linter